gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/util/ThrowableResources.java

```diff
@@ -25,6 +25,14 @@ public final class ThrowableResources {
     public static final String CAUSAL_CHAIN = "* Causal chain is:";
     public static final String EXCEPTION_MESSAGE = "* Full exception is:";
 
+    public static final String FOOTER =
+            "This failure report is produced by the `com.palantir.failure-reports plugin`, which retrieves information"
+                + " from the exceptions generated by failed tasks. For further details about the failure, refer to the"
+                + " stdout logs. Due to a known issue (https://github.com/gradle/gradle/issues/6068), error logs for a"
+                + " generic task are not included in the report.\n"
+                + "Please report any issues and share your feedback at"
+                + " https://github.com/palantir/gradle-failure-reports/issues.";
```

Comment:
I don't think we need this, it's just going to increase the verbosity for every exception and possibly cause people to make issues on this repo for failures from unrelated repos. I don't think having stdout/stderr from tasks is something I would expect for general tasks - ideally if something went wrong they should put it in the exception. If we're executing an external command that does have stdout/stderr, I'd expect that exec task to put the stdout/err into the exception.

Reply:
hmm, my concern is that if the exception doesn't contain the stdout/stderr, the report would actually not be very useful and it might take more time for people to check the full CircleCi logs. eg. I tested with a conjure invalid definition and the  conjure exception did not contain the stdOutput error message. I'll have a look if this can be easily changed. 

I'll remove this though, makes sense to not have it for every exception.